recipes/python-configuration/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "python-configuration" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 63564900eb147a304e3b8f2c9d0ebfe0438276c23282441ab7e63fee2896a0fc
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - poetry
+  run:
+    - python
```

`host` and `run` should almost always have the same constraint:
```suggestion
    - python >=3.6
```